openslides/core/static/js/core/base.js

```diff
@@ -852,48 +852,94 @@ angular.module('OpenSlidesApp.core', [
     }
 ])
 
+/* Two functions to convert between time duration in seconds <-> human readable time span.
+ * E.g. 90 sec <-> 1:30 (min), 3661 sec <-> 1:01:01 (h)
+ *
+ * secondsToHumanTime: Expects seconds and give [h*:]mm[:ss]. The minutes part is always given, the hours
+ *      and minutes could be controlled. The default are forced seconds and hours just if it is not 0.
+ *      - forceSeconds (default true): show seconds even if 00
```

Comment:
If I understand this correctly we have 3 cases. Always seconds, only seconds if they are greater than 0, no seconds.

Am I right? If yes please implement this using only one option flag (accepting strings like 'enable', 'auto', 'disable' or something like this) and not two bool flag.